lib/ansible/modules/remote_management/ucs/ucs_dns_server.py

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = r'''
+---
+module: ucs_dns_server
+short_description: Configures DNS server on Cisco UCS Manager
+extends_documentation_fragment:
+- ucs
+description:
+- Configures DNS server on Cisco UCS Manager.
+- Examples can be used with the L(UCS Platform Emulator,https://communities.cisco.com/ucspe).
+options:
+  state:
+    description:
+    - If C(absent), will remove a DNS server.
+    - If C(present), will add or update a DNS server.
+    choices: [absent, present]
+    default: present
+
+  dns_server:
+    description:
+    - DNS server IP address.
+    - Enter a valid IPV4 Address.
+    aliases: [ name ]
+    default: ""
```

Comment:
Why do we have `default: ""` here ? The default is `None` but that doesn't have to be exposed.
I would leave the default out of this.

Reply:
I have removed this, makes sense.